Source/Core/DolphinWX/ControllerConfigDiag.cpp

```diff
@@ -37,6 +38,20 @@
 #include "DolphinWX/X11Utils.h"
 #endif
 
+static wxButton* CreateButtonWithMinSize(wxWindow* parent, wxWindowID id, const wxString& label, wxSize size)
```

Comment:
It may be beneficial to move this to WxUtils.cpp/.h, since this can also be used elsewhere as well.